apps/src/codebridge/Editor/index.tsx

```diff
@@ -38,6 +38,15 @@ export const Editor = ({langMapping, editableFileTypes}: EditorProps) => {
     }
   }, [file?.language, langMapping]);
 
+  if (file && viewableImageFileType(file.language)) {
+    const base64 = window.btoa(file.contents);
+    return (
+      <div>
+        <img src={`data:image/png;base64,${base64}`} alt={file.name} />
+      </div>
+    );
+  }
+
```

Comment:
simple viewer - if an image file is selected (gif, jpg, png), then just display an image.

Reply:
Do we want to allow image uploads? Do programs still run if there's an image file in the project?

Reply:
Oh. Do we not? I must've been thinking of weblab. Locked it down so it's only text types, but will confirm with Sam et. al.